### DISCUSSION TITLE
Drive integration

### DISCUSSION THREAD

**Discussion by PChild** [team401/high-key-2024#25] [General]
Tested drive, need to debug, one of the wheels spins in place with nothing commanded. Also should check kinematics initialization,  I readded that on one of the crappy team laptops and it may be wrong.

**Comment by aidnem** [comment]
@minhnguyenbhs I had to swap laptops because yours had Phoenix Tuner X installed on it and the windows store app wasn't working on mine. I forgot to swap accounts so it looks like you got attributed for a couple of pushes. Sorry about that!
